packages/components/order-tracking/src/index.js

```diff
@@ -0,0 +1,20 @@
+import angular from 'angular';
+import translate from 'angular-translate';
```

Comment:
Related to a peer dependency we tend to use:

```suggestion
import 'angular-translate';
```